pkg/aggregator/aggregator.go

```diff
@@ -261,10 +262,12 @@ type BufferedAggregator struct {
 	health    *health.Handle
 	agentName string // Name of the agent for telemetry metrics
 
-	tlmContainerTagsEnabled bool                                         // Whether we should call the tagger to tag agent telemetry metrics
-	agentTags               func(types.TagCardinality) ([]string, error) // This function gets the agent tags from the tagger (defined as a struct field to ease testing)
-	globalTags              func(types.TagCardinality) ([]string, error) // This function gets global tags from the tagger when host tags are not available
-
+	tlmContainerTagsEnabled     bool                                         // Whether we should call the tagger to tag agent telemetry metrics
+	agentTags                   func(types.TagCardinality) ([]string, error) // This function gets the agent tags from the tagger (defined as a struct field to ease testing)
+	globalTags                  func(types.TagCardinality) ([]string, error) // This function gets global tags from the tagger when host tags are not available
+	enrichTags                  func(tagset.TagsAccumulator, taggertypes.OriginInfo)
+	checksCardinality           func() types.TagCardinality
+	tagger                      tagger.Component
```

Comment:
Maybe add a couple blank lines to avoid shifting types and comments everything so far right.

```suggestion
	tlmContainerTagsEnabled     bool                                         // Whether we should call the tagger to tag agent telemetry metrics

	agentTags                   func(types.TagCardinality) ([]string, error) // This function gets the agent tags from the tagger (defined as a struct field to ease testing)
	globalTags                  func(types.TagCardinality) ([]string, error) // This function gets global tags from the tagger when host tags are not available
	enrichTags                  func(tagset.TagsAccumulator, taggertypes.OriginInfo)
	checksCardinality           func() types.TagCardinality
	tagger                      tagger.Component

```